modules/ckanext-ytp_main/ckanext/ytp/plugin.py

```diff
@@ -264,7 +264,7 @@ def dataset_facets(self, facets_dict, package_type):
         lang = get_lang_prefix()
         facets_dict = OrderedDict()
         facets_dict.update({'vocab_international_benchmarks': _('International Benchmarks')})
-        facets_dict.update({'collection_type': _('Collection Types')})
+        facets_dict.update({'collection_type': _('Apiset') if package_type == 'apiset' else _('Collection Types')})
```

Comment:
Why is this changed ? The design has different facet filters anyway, so this change does not seem to make sense.

Reply:
It seems to me, the views in "/dataset" and "/apiset" use the same "Collection Types" translation, which translates to "Dataset", which is wrong in "/apiset" view.

Reply:
`collection_type` does not exist in the schema of apis, so it would be need to be removed all together. We will update the filters separately as they will be different from datasets, something like harvest does https://github.com/ckan/ckanext-harvest/blob/8874175bc9c09c40ad822df2101ce14fb957f58f/ckanext/harvest/plugin/__init__.py#L340-L347

Reply:
Do you mean I should keep that line as it was? Removing it will not display the translation for datasets either I suppose.

Reply:
This change is not needed, yes the UI says Tietoaineisto but this filters will be reimplemented anyhow.

Reply:
Removed the change